predicators/pybullet_helpers/link.py

```diff
@@ -60,3 +60,13 @@ def get_link_pose(body: int, link: int, physics_client_id: int) -> Pose:
 
     link_state = get_link_state(body, link, physics_client_id)
     return link_state.pose
+
+
+def get_relative_link_pose(body: int, link1: int, link2: int,
+                           physics_client_id: int) -> Pose:
+    """Get the pose of one link relative to another link on the same body."""
```

Comment:
Update comment to specify which link - i.e. pose of link 1 relative to the frame of link 2